packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -353,6 +353,7 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                             size: blob.byteLength,
                             serverRetries: res.headers.get("x-fluid-retries") ?? undefined,
                             waitQueueLength: this.epochTracker.rateLimiter.waitQueueLength,
+                            cacheControl: res.headers.get("cache-control") ?? undefined,
```

Comment:
should we be more explicit in our intentions? I.e. logging it like that requires diligence to look at the data / notice when things are going wrong. If we believe that all the data should be always cached, then I'd rather prefer an error event when it's not.

Reply:
Added the error event.